### ISSUE TITLE
Problem scraping Bet365 with headless mode

### ISSUE THREAD

**Issue by victornoleto** [S1M0N38/soccerapi#38]
I know that my question is not exactly related to the repository, but I thought it was worth sharing.

I created a thread on stackoverflow and there extremely detailed what my problem is when web scraping the Bet365 site.

Thanks if you can take a look.

[https://stackoverflow.com/questions/70883112/problem-scraping-bet365-with-headless-mode-python-selenium-puppeteer](https://stackoverflow.com/questions/70883112/problem-scraping-bet365-with-headless-mode-python-selenium-puppeteer)

**Comment by S1M0N38** [comment]
Have you already tried [soccerapi-server](https://github.com/S1M0N38/soccerapi-server) ?

**Comment by victornoleto** [comment]
Isn't this server just for generating a new `X-Net-Sync-Term`?

**Comment by S1M0N38** [comment]
Yes, but you can use the same technology (puppeteer noheadless inside docker container with virtual display) to achieve your goals. Moreover the stealth plugin improve anti-bot detection

**Comment by Colombo97** [comment]
@victornoleto Did u solve the issue?

**Comment by HMaker** [comment]
In last changes bet365 tries to detect headless chrome but [puppeteer-stealth](https://github.com/berstend/puppeteer-extra/tree/master/packages/puppeteer-extra-plugin-stealth) is able to bypass their naive detection.

**Comment by Colombo97** [comment]
@HMaker Thank you for feedback, i'll try it

**Comment by victornoleto** [comment]
Unfortunately I gave up on finding a solution. I didn't get to test the solution proposed by @HMaker.

**Comment by Colombo97** [comment]
Lets dicuss about it add me on discord: Colombo97#9666
